app/confirm-email/page.tsx

```diff
@@ -0,0 +1,26 @@
+import { redirect } from 'next/navigation';
+
```

Comment:
server-side page that will call the backend to verify the userId and token from the email link